extensions/sql-database-projects/src/test/autorestHelper.test.ts

```diff
@@ -25,6 +25,10 @@ describe('Autorest tests', function (): void {
 		sinon.restore();
 	});
 
+	after(() => {
+		testUtils.deleteGeneratedTestFolder();
```

Comment:
the PR validation check is failing because all these calls to `testUtils.deleteGeneratedTestFolder()` either need an await or void, now that the no-floating-promises rule is on in this extension (added in https://github.com/microsoft/azuredatastudio/pull/16943)

Reply:
Updated them.